File: Iniciante/1010.js
let line1 = lines.shift().split(' ');
let line2 = lines.shift().split(' ');

let cod1 = Number(line1.shift());
let np1 = Number(line1.shift());
let vu1 = Number(line1.shift());

let cod2 = Number(line2.shift());
let np2 = Number(line2.shift());
let vu2 = Number(line2.shift());

let valor = np1*vu1 + np2*vu2;

console.log("VALOR A PAGAR: R$ " + valor.toFixed(2));

File: Iniciante/1010.py
line1 = input().split()
line2 = input().split()

c1 = int(line1[0])
n1 = int(line1[1])
v1 = float(line1[2])

c2 = int(line2[0])
n2 = int(line2[1])
v2 = float(line2[2])

calculo = n1*v1+n2*v2

print("VALOR A PAGAR: R$ {:.2f}".format(calculo))
